src/main/java/at/hannibal2/skyhanni/features/inventory/caketracker/CakeTracker.kt

```diff
@@ -350,56 +350,52 @@ object CakeTracker {
         lastKnownCakeDataHash = 0
     }
 
-    private fun buildDisplayTypeToggle(): Renderable = Renderable.horizontalContainer(
-        buildList {
-            val ownedColor = if (config.displayType == DisplayType.OWNED_CAKES) "§a" else "§e"
-            val missingColor = if (config.displayType == DisplayType.MISSING_CAKES) "§a" else "§e"
-
-            add(
-                Renderable.optionalLink(
-                    "$ownedColor[Owned]",
-                    { setDisplayType(DisplayType.OWNED_CAKES) },
-                    condition = { config.displayType != DisplayType.OWNED_CAKES },
-                ),
-            )
-            add(Renderable.string(" "))
-            add(
-                Renderable.optionalLink(
-                    "$missingColor[Missing]",
-                    { setDisplayType(DisplayType.MISSING_CAKES) },
-                    condition = { config.displayType != DisplayType.MISSING_CAKES },
-                ),
-            )
-        },
-    )
+    private fun buildDisplayTypeToggle(): Renderable = Renderable.line {
+        val ownedColor = if (config.displayType == DisplayType.OWNED_CAKES) "§a" else "§e"
+        val missingColor = if (config.displayType == DisplayType.MISSING_CAKES) "§a" else "§e"
+
+        add(
+            Renderable.optionalLink(
+                "$ownedColor[Owned]",
+                { setDisplayType(DisplayType.OWNED_CAKES) },
+                condition = { config.displayType != DisplayType.OWNED_CAKES },
+            ),
+        )
+        add(Renderable.string(" "))
```

Comment:
```suggestion
        addString(" ")
```
maybe these could be replaced by a addSeperator or smth idk